emulator/emulator_factory.go

```diff
@@ -5,28 +5,17 @@ import (
 	"pm5-emulator/sm"
 
 	"github.com/bettercap/gatt"
-	"github.com/bettercap/gatt/linux/cmd"
+	"github.com/bettercap/gatt/examples/option"
 )
 
 //NewEmulator factory methods initializes emulator
 func NewEmulator() *Emulator {
-	d, err := gatt.NewDevice(defaultServerOptions...)
+	d, err := gatt.NewDevice(option.DefaultServerOptions...)
```

Comment:
I'd say let's not use it directly from examples. How about checking the GOOS in runtime? 

```
var defaultServerOptions=func() []gatt.Options{
  //check the runtime.GOOS here and return the options accordingly
}()

Reply:
Runtime feels a bit strange when this already works at compile time. Like why include code in macos binaries that will never be run

Lets make our own module called `options` to store the platform specific defaults

Reply:
>Runtime feels a bit strange when this already works at compile time. Like why include code in macos binaries that will never be run

Yep, that sounds good! Completely missed the OS specific suffixes idea in Go. 

>Lets make our own module called options to store the platform specific defaults

Lets go with this ⬆️

Reply:
Sounds good to me, i might not be able to get to this today, so if you have time, have at it!